provider/k8s/process.go

```diff
@@ -285,24 +286,28 @@ func (p *Provider) ProcessRun(app, service string, opts structs.ProcessRunOption
 		release = a.Release
 	}
 
-	pd, err := p.Cluster.CoreV1().Pods(p.AppNamespace(app)).Create(&ac.Pod{
-		ObjectMeta: am.ObjectMeta{
-			Annotations: map[string]string{
-				// "iam.amazonaws.com/role": ns.ObjectMeta.Annotations["convox.aws.role"],
-			},
-			GenerateName: fmt.Sprintf("%s-", service),
-			Labels: map[string]string{
-				"app":     app,
-				"rack":    p.Name,
-				"release": release,
-				"service": service,
-				"system":  "convox",
-				"type":    "process",
-				"name":    service,
+	pd, err := p.Cluster.CoreV1().Pods(p.AppNamespace(app)).Create(
+		context.TODO(),
+		&ac.Pod{
+			ObjectMeta: am.ObjectMeta{
+				Annotations: map[string]string{
+					// "iam.amazonaws.com/role": ns.ObjectMeta.Annotations["convox.aws.role"],
+				},
```

Comment:
Can we delete this annotation?

Reply:
Yeah, I'm actually working on this file with the docker/buildkit replacement and I removed the line.
Should be on my next PR.